Cargo.toml

```diff
@@ -4,7 +4,6 @@ version = "0.6.2"
 edition = "2021"
```

Comment:
@rodneykinney can u bump version to `0.6.3` in `Cargo.toml` and `pyproject.toml`? So we can make a new pypi release.